src/core/components/SiteNotice/index.js

```diff
@@ -0,0 +1,58 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import { sanitizeHTML } from 'core/utils';
+import Notice from 'ui/components/Notice';
+import type { I18nType } from 'core/types/i18n';
+import type { AppState } from 'amo/store';
+
+import './styles.scss';
+
+type Props = {|
+  i18n: I18nType,
+  siteIsReadOnly: boolean,
+  siteNotice: string | null,
+|};
+
+export class SiteNoticeBase extends React.Component<Props> {
```

Comment:
If you named this component `SiteNotices` then it would provide a hint that it renders a list of components, which could be helpful to someone.